tests/E2E Tests/WebAppUiTests/TestingWebAppLocally.cs

```diff
@@ -103,5 +103,86 @@ public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailPassword()
             playwright.Dispose();
         }
     }
+
+    [Fact]
+    [SupportedOSPlatform("windows")]
+    public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailWithCiamPassword()
+    {
+        // Arrange
+        const string _clientSecret = "...";
+        Process? process = null;
+        const string TraceFileName = TraceFileClassName + "_ValidEmailPassword";
+        using IPlaywright playwright = await Playwright.CreateAsync();
+        IBrowser browser = await playwright.Chromium.LaunchAsync(new() { Headless = false });
+        IBrowserContext context = await browser.NewContextAsync(new BrowserNewContextOptions { IgnoreHTTPSErrors = true });
+        await context.Tracing.StartAsync(new() { Screenshots = true, Snapshots = true, Sources = true });
+
+        try
+        {
+            var clientEnvVars = new Dictionary<string, string>
+            {
+                {"AzureAd__ClientId", "16ee75bf-ebbe-42a8-9474-caa0fe856bc7"},//"Authority": "https://nativeauthasampleapp.ciamlogin.com"
+                {"AzureAd__Instance", "https://nativeauthasampleapp.ciamlogin.com"},
```

Comment:
Same as above, expecting the Authority to be set (for both CUD and non-CUD). Instance + TenantID not expected to work.